posts/navarroaxel/16.git-autocorrect.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Git autocorrect needs more marketing
+published: true
+description: How to configure Git to autofix typos in your input commands
+tags: 'git, bash, productivity, tooling'
+cover_image: ./assets/git-autocorrect.png
+id: 1905594
+---
+
+Git comes with a lot of configurations to improve your experience with the command-line. We already talked about how to [color moved blocks of code](https://dev.to/cloudx/how-to-color-the-moved-code-in-git-10ei), but recently I found an awesome configuration called [autocorrect](https://git-scm.com/docs/git-config#Documentation/git-config.txt-helpautoCorrect). 🤩
+
+This feature allows Git to detect typos in your input and display similar valid commands. Git can also suggest the correct one or even run the suggestion automatically. Let's see!
+
+## The default value
+
+By default Git displays suggested commands when we had a typo:
+
+```bash
+$ git pll
+git: 'pll' is not a git command. See 'git --help'.
+
+The most similar command is
+    	pull
+```
+
+But we can get more than just a suggestion… 👀
```

Comment:
I assume you mean more information, not more than one suggestion. If that's the case this is fine